experimenter/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.tsx

```diff
@@ -137,21 +137,29 @@ const TableAudience = ({ experiment }: TableAudienceProps) => {
                 {experiment.isSticky ? "True" : "False"}
               </td>
 
-              <th>First Run Experiment</th>
-              <td data-testid="experiment-is-first-run">
-                {experiment.isFirstRun ? "True" : "False"}
-              </td>
-            </tr>
-            <tr>
-              <th>First Run Release Date</th>
-              <td colSpan={3} data-testid="experiment-release-date">
-                {experiment.proposedReleaseDate ? (
-                  experiment.proposedReleaseDate
-                ) : (
-                  <NotSet color="primary" />
-                )}
-              </td>
+              {!isDesktop ? (
```

Comment:
This is fine for now, but at some point we'll want to update this to be checking exclusively for mobile applications once we have Pocket etc on Nimbus.

Reply:
Good point, we're using this `!isDesktop` check in a few different places. I just filed a ticket for this so that we don't forget to switch this over: https://github.com/mozilla/experimenter/issues/8974

Reply:
Yeah a pattern we've used in the past is to introduce something like

`shouldHaveReleaseDate`

and then you define that however based on other predicates